<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditReadDialog</class>
 <widget class="QDialog" name="EditReadDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>388</width>
    <height>282</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>290</width>
    <height>245</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>100000</width>
    <height>100000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Element Filter</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>220</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditFilter">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>55</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Filter</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxMatchCase">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Match Case</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditNamePrefix">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>110</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Name Prefix [b,q,s, ...]</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Scale [result = data*scale]</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxScale">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>180</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxCompareOnly">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Compare Only</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>55</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>File Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditFileName">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>30</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditReadDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditReadDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>EditReadDialog</receiver>
   <slot>selectFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>selectFile()</slot>
 </slots>
</ui>
